src/components/Form/ServiceForm.tsx

```diff
@@ -98,32 +101,41 @@ function ServiceForm() {
           token.decimals,
         );
         const parsedRateAmountString = parsedRateAmount.toString();
-        const cid = await postToIPFS(
-          JSON.stringify({
+
+        let tx, cid;
+
+        cid = await tlClient?.service?.updloadServiceDataToIpfs(
+          {
             title: values.title,
             about: values.about,
             keywords: values.keywords,
             rateToken: values.rateToken,
             rateAmount: parsedRateAmountString,
-          }),
+          },
         );
 
-        // Get platform signature
-        const signature = await getServiceSignature({ profileId: Number(user?.id), cid });
-
-        let tx;
-
         if (isActiveDelegate) {
           const response = await delegateCreateService(chainId, user.id, user.address, cid);
           tx = response.data.transaction;
         } else {
-          tx = await walletClient.writeContract({
-            address: config.contracts.serviceRegistry,
-            abi: ServiceRegistry.abi,
-            functionName: 'createService',
-            args: [user?.id, process.env.NEXT_PUBLIC_PLATFORM_ID, cid, signature],
-            account: address,
-          });
+
+          if (tlClient) {
+            console.log("Starter kit: creating new service")
+            const serviceResponse = await tlClient.service.create({
+              title: values.title,
+              about: values.about,
+              keywords: values.keywords,
+              rateToken: values.rateToken,
+              rateAmount: parsedRateAmountString
+            }, user.id, 4);
```

Comment:
easier since first dev to directly put env variable to prevent any issue or confusion, here replace 4 process.env.NEXT_PUBLIC_PLATFORM_ID